webpage has some styling and organized content

HelioStrike opened the pull request:
@skashish-sps Sorry, I got a bit carried away.

I added a bit of styling, added some images, added scrolling on key presses (which is weird considering you could press down arrow for that, but it makes sense in context).

skashish-sps commented:
Is there somewhere I can check your website

skashish-sps commented on portfolio/src/main/webapp/css/semantic.min.css:
```diff
@@ -0,0 +1,372 @@
+ /*
```
Why copy the entire file? I suggest use a CDN based import for all libraries you are using. Code duplication is something that is pointless and unnecessary

HelioStrike replied:
Somehow, I was under the assumption that serving this way was better. I'll fix that.

skashish-sps commented:
So I did look at the webpage you created.
Good effort to start off. I have a few suggestions.

- Are the doodles and other image assets you put up, your own creation? If not, can you use just links instead of copying files as assets. Also protects, against copyright.

- The same goes for css and fonts files, If you have just copied the files, like semantic-ui, you can just use cdn to reference the files at the correct version. This will reduce your repository size by quite a bit. Secondly, it will make code reviews easier.

- The scroll on any key, I seem to be missing something, there is no way for me to go back up. Except the existing up arrow, pressing other keys to scroll down proves counter intuitive, would not suggest put it. I appreciate the effort, but it is something that makes the experience worse.

- Nit: The size of logos of github and linkedin, seem to large for a good impactful portfolio project. Can we reduce the size?

HelioStrike commented:
@skashish-sps 

I will try to see if I can host the page somewhere. The easiest way to view it right now is to pull and open the index.html file.

- The doodles are my own. I removed the video game images. Now that I look at it, it feels a little odd to list them out.
- I downloaded the font assets (free to use) as a zip from [1001fonts](https://www.1001fonts.com/).
- Yea, removed it. I initially planned on making a carousel, where this would've worked, but gave up after spending 2 hrs trying to center the carousel's content.
- They're 32px now (were 60px earlier). What do you think?

skashish-sps commented:
1. Fonts look fine.
2. 32px is ok
3. Github has its own HTML hosting : https://htmlpreview.github.io/?https://github.com/HelioStrike/sps/blob/0a04b6daa493356fbe04a4323d431378704bb804/portfolio/src/main/webapp/index.html

skashish-sps commented on portfolio/.classpath:
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
.classpath and all are not necessary.
Please create a gitignore file and all system folders to github

skashish-sps replied:
You can refer this for a comprehensive list of items in gitignore
https://gist.github.com/jacks0n/cc0386e71d30c66a670e
or this: https://github.com/github/gitignore

skashish-sps commented on portfolio/.project:
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
remove this file from VCS

skashish-sps commented on portfolio/.settings/org.eclipse.core.resources.prefs:
```diff
@@ -0,0 +1,3 @@
+eclipse.preferences.version=1
```
remove this file from VCS

skashish-sps commented on portfolio/.settings/org.eclipse.jdt.apt.core.prefs:
```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```
remove this file from VCS

skashish-sps commented on portfolio/.settings/org.eclipse.jdt.core.prefs:
```diff
@@ -0,0 +1,9 @@
+eclipse.preferences.version=1
```
remove this file from VCS

skashish-sps commented on portfolio/.settings/org.eclipse.m2e.core.prefs:
```diff
@@ -0,0 +1,4 @@
+activeProfiles=
```
remove this file from VCS

HelioStrike commented:
Made the changes (updated .gitignore).

HelioStrike commented:
@skashish-sps I missed the instructions describing the deployment procedure. The page can be viewed [here](https://smutyala-sps-summer20.el.r.appspot.com) too now.

HelioStrike committed:
webpage has some styling and organized content

skashish-sps reviewed (approved):
LGTM
